src/main/java/com/efarm/efarmbackend/service/user/UserService.java

```diff
@@ -135,6 +138,67 @@ public List<UserSummaryDTO> getActiveFarmUsersByFarmId() {
                 .collect(Collectors.toList());
     }
 
+    @Transactional
+    public void toggleUserActiveStatus(Integer userId) {
+        User user = userRepository.findById(Long.valueOf(userId))
```

Comment:
jak mialbym tez sie czepiac to error message jest nie dokladne bo racja nie ma uzytkownika o takim id na tej farmie ale dokladniejszym by bylo ze wogole nie istnieje i wtedy mozna pomyslec ze to by bylo trzeba inaczej zrobic moze findByIdAndFarm() w repozytorium. Error message takie jak id nie istnieje to w sumie troche slabe dla rolnikow bo jak im powiesz ze np. sprzet o id 25 nie istnieje to nie sadze ze beda wiedzieli o co chodzi a jak napiszesz Zetor 7211 nie istnieje to troche lepiej ale to do twojej opini zostawie czy trzeba zmienic. tutaj by bylo ze uzytkownik o loginie abcd nie istnieje w tym przypadku